src/components/TodosHeader/TodosHeader.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+type Props = {};
+
+export const TodosHeader: React.FC<Props> = props => {
+  const {} = props;
```

Comment:
let's remove this empty destructuring

Reply:
it`s a billet
removed

Reply:
there are comments about nothing
everything that was removed was added and expanded in the 2nd part of the solution
remove and add again
monkey work